app/src/main/java/org/simple/clinic/contactpatient/PatientDetails.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.contactpatient
+
+import org.simple.clinic.medicalhistory.Answer
+import org.simple.clinic.patient.Gender
+import java.time.Instant
+
+data class PatientDetails(
+    val name: String,
+    val gender: Gender,
+    val age: Int,
+    val phoneNumber: String?,
+    val patientAddress: String,
```

Comment:
Can we confirm if the patient address is not nullable?

Reply:
Yeah, it's not nullable in `OverdueAppointment`